org.gtkhash.gtkhash.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: org.gtkhash.gtkhash
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: gtkhash
+
+rename-desktop-file: gtkhash.desktop
+rename-appdata-file: gtkhash.appdata.xml
+rename-icon: gtkhash
+copy-icon: true # GtkAboutDialog uses logo-icon-name:PACKAGE
```

Comment:
You should rename those upstream instead

Reply:
Will do for next release.